packages/main/src/FileUploader.ts

```diff
@@ -317,6 +357,28 @@ class FileUploader extends UI5Element implements IFormInputElement {
 		}, "");
 	}
 
+	/**
+	 * Returns true if all files are below the maxFileSize, otherwise throws a "fileSizeExceeded" event if one file exceeds it.
+	 * @private
+	 */
+	_checkFileSize(files: FileList | null): boolean {
+		if (!(this.maxFileSize && files)) {
```

Comment:
Can we transform this using De Morgans law? IMO it's easier to read that way

Reply:
You want it expanded? Sure :)